src/Form/Fields.php

```diff
@@ -206,6 +216,8 @@ public static function for(
 
```

Comment:
`Suggestion:` For simplicity, maybe pass here `$language` simply to the constructor which now already takes care of ensuring the language object.